### ISSUE TITLE
Toolkit incorrectly marks disabled button as having an insufficient color contrast error

### ISSUE THREAD

**Issue by kirbykatie** [ThePacielloGroup/ARC-Toolkit#50]
**Describe the bug**

Running the toolkit on a page with a disabled button, the toolkit shows an error and identifies that button as not having enough color contrast (the button is light gray with white text).

Sufficient color contrast is not needed for disabled elements, according to [WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

**To Reproduce**

> Identify any steps required to reproduce the behavior and/or link to a test case that demonstrates the issue.

1. Create or find a webpage with the following markup:
`<button disabled>Test</button>`
and with the following styling:
`button:disabled {
    background-color: #e8e5e1;
    color: #ffffff;
}`
2. Run the ARC toolkit to confirm error.

**Expected behavior**

There should be no error regarding color contrast for disabled buttons.

**Screenshots**

![disabled-error](https://github.com/ThePacielloGroup/ARC-Toolkit/assets/41655241/20e5c64c-72a7-4c50-b145-c348ed9755a3)

Markup of button:
![markup](https://github.com/ThePacielloGroup/ARC-Toolkit/assets/41655241/da7f640f-7b0d-4423-b0bc-56e932877649)

**Version information**

- Browser and version: Chrome 120.0.6099.71
- ARC Toolkit version: 5.6.0

**Additional context**
> Add any other context about the problem here.

**Comment by aaronbnb** [comment]
Thank you @kirbykatie this is a great bug report and a bug that is correctable via automated means.

**Comment by kirbykatie** [comment]
Thank you for looking into it! I also just learned another team we're working with found the same issue with a disabled checkbox, so I just want to flag that all disabled HTML elements should not get an insufficient color contrast error.

**Comment by kirbykatie** [comment]
Hello! Following up on this - is there a an upcoming release where this issue is expected to be resolved?

**Comment by ferllings** [comment]
Sorry for the delay. The next release will include this fix.
